riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb

```diff
@@ -0,0 +1,377 @@
+{
```

Comment:
<p>Should OSOM be all CAPs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ridatadiscoverycenter/riddc-jbook/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I changed the syntax to OSOM.